tests/numpy/test_accuracy_extreme_values.py

```diff
@@ -52,10 +52,10 @@ def test_norm(self) -> bool:
         logger.error("TestAccuracyXtremeNorm.test_norm()")
         logger.info("Normal noise model not implemented for numpy")
 
-        # np.random.seed(1)
-        # self._test_low_values(noise_model="norm")
-        # self._test_zero_variance(noise_model="norm")
-        return True
+        np.random.seed(1)
+        ret_val = self._test_low_values(noise_model="norm")
+        np.random.seed(1)
+        return ret_val and self._test_zero_variance(noise_model="nb")
```

Comment:
the tests shouldn't actually have a return value but instead `assert False` if the fail.

Reply:
yes do that please, I removed the returns for the nb related stuff already on my branches

Reply:
@picciama I have had a look and I simply think checking whether the model actually fits or not is the only goal.   The underlying class seems to be just from the `test_accuracy.py` file, which looks at accuracy.  But at the top of this file, it pretty clearly states "Test whether numerical extremes throw error in initialisation or during first training steps." and then the subsequent classes say something like "Test whether optimizers yield exact results for Poisson distributed data." which seems to be at odds with the goal of the top (or even be a goal that makes sense - these extreme value tests shouldn't be fitting anything, right?)

Reply:
True, that is misleading. I had a look at it. My suggestion would be to test for initialisation as well as accuracy after training. Accuracy used to be a problem as we had parameter clipping margins that prevented the model from being accurate enough in case for very small or large counts. I removed them for that reason / let them fail. Since I found a way around this while implementing edgeR, we can readd the accuracy testing at a later point after fixing the parameter clipping, since this needs thorough testing as well (should be done on separate branch). 
Therefore, you can leave it like that for now. We can also replace the return with assert later on, this isn't to big of an issue.